blog/README.md

```diff
@@ -0,0 +1,259 @@
+# CoreOS Layaring
+
+In this blog we are going to explore the coreos-layering technology in order to
+inject kernel-modules to an Openshift node.
+
```

Comment:
```suggestion
# Customizable RHEL CoreOS
Some OpenShift administrators asked for more flexibility for RHEL CoreOS customization in specific contexts. The RHCOS image layering feature is generally available in OpenShift 4.13. We can now create a customized base image using a different kernel or any particular software and consume it in a given cluster.

RHCOS image layering allows you to easily extend the functionality of your base image by layering additional images onto the base image. This layering does not modify the base image, and it creates a custom layered image that includes all base functionality and adds one to specific nodes in the cluster.
```

Reply:
Thanks.
I have inserted that introduction.